string name

float32 wheel_diameter
